File: app/src/main/java/com/lucagiorgetti/surprix/ui/mainfragments/surpriseList/MissingRecyclerAdapterListener.java
package com.lucagiorgetti.surprix.ui.mainfragments.surpriseList;

import com.lucagiorgetti.surprix.model.Surprise;

public interface MissingRecyclerAdapterListener extends BaseSurpriseRecyclerAdapterListener{
    void onShowMissingOwnerClick(Surprise surprise);
}

File: app/src/main/java/com/lucagiorgetti/surprix/ui/mainfragments/surpriseList/MissingRecyclerAdapterListener.kt
package com.lucagiorgetti.surprix.ui.mainfragments.surpriseList

import com.lucagiorgetti.surprix.model.Surprise

interface MissingRecyclerAdapterListener : BaseSurpriseRecyclerAdapterListener {
    fun onShowMissingOwnerClick(surprise: Surprise)
}